src/vimscript/parserUtils.ts

```diff
@@ -113,7 +113,7 @@ const escapedParser = string('\\')
 export const keystrokesExpressionParser: Parser<string[]> = alt(
   escapedParser,
   specialCharacterParser,
-  noneOf('"'),
+  regexp(/./),
```

Comment:
This should work, right?

```suggestion
  regexp(any),
```

Reply:
Yes, that should work!
The main difference is that `any` can parse raw special characters like `\n`, whereas `/./` cannot.
However, in this plugin, the input keystrokes are not expected to include raw special characters such as `\t` or `\n`. These should always be escaped as `\\t` or `\\n`, in which case they are handled by escapedParser.

So as long as the plugin is working as intended and only receives properly escaped input, keystrokesExpressionParser should work correctly with either option.
However, if raw special characters are accidentally included, the behavior would differ depending on which parser is used.

Personally, I think using `/./` is safer, since if any raw special character slips in, it simply won't be parsed or included in the result, making it easier to notice unexpected input.

Reply:
Got it, thanks!